src/parser/rogue/outlaw/modules/talents/BladeRush.tsx

```diff
@@ -21,20 +21,23 @@ class BladeRush extends Analyzer {
   }
 
   onFinishMove(event: CastEvent) {
-    if (!getResource(event.classResources, RESOURCE_TYPES.COMBO_POINTS.id)) {
+    if (!event.classResources || !getResource(event.classResources!, RESOURCE_TYPES.COMBO_POINTS.id)) {
```

Comment:
```suggestion
    if (!event.classResources || !getResource(event.classResources, RESOURCE_TYPES.COMBO_POINTS.id)) {
```

The first check should already tell TS that event.classResources now no longer can be undefined.

Reply:
Good catch. I need to get out if JS mode and into TS mode to catch things like this when reviewing.